wheelhouse.txt

```diff
@@ -0,0 +1,2 @@
+calver
+hatchling
```

Comment:
sorry, why are these packages included?  Neither are used.
```suggestion
```

Reply:
Okay, I resolved an issue in the layer-docker which was requiring these packages.  Please try without them

Reply:
Ah, I was pretty sure I needed them for something. I've removed them now, thanks for fixing the root cause!